connector/authproxy/authproxy.go

```diff
@@ -37,7 +37,7 @@ type callback struct {
 }
 
 // LoginURL returns the URL to redirect the user to login with.
-func (m *callback) LoginURL(s connector.Scopes, callbackURL, state string) (string, error) {
+func (m *callback) LoginURL(s connector.Scopes, callbackURL, state string, _ map[string]string) (string, error) {
```

Comment:
nit: would it make sense to use `map[string]string` instead of `url.Values` here, for consistency? (or, conversely, would it make sense to make the other signatures that ignore this argument accept `map[string]string` as well?)